Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ControlSchemesView.cs

```diff
@@ -31,8 +31,21 @@ public ControlSchemesView(VisualElement root, StateContainer stateContainer, boo
             {
                 Dispatch((in InputActionsEditorState state) =>
                 {
-                    m_NewName = ControlSchemeCommands.MakeUniqueControlSchemeName(state,
-                        ((TextField)evt.currentTarget).value);
+                    // If the name is the same as the current name, don't change it
+                    var newName = ((TextField)evt.currentTarget).value.Trim();
+                    if (string.IsNullOrEmpty(newName) || String.Compare(newName, state.selectedControlScheme.name) == 0)
```

Comment:
Minor: I think you can remove the string.IsNullOrEmpty since it cannot be null from Trim() right?
In addition String.Compare will be 0 if newName is empty.

Reply:
yes it can't be null but IsEmpty doesn't exist.
String.Compare() will not be 0 if the string is empty, except if both are empty